File: Frontend/trackr-paciente/src/app/shared/Dtos/notificaciones/notificacion-paciente-popover-dto.ts
export class NotificacionPacientePopOverDto
{
    public idTipoNotificacion : number
    public id: number;
    public titulo: string;
    public mensaje: string;
    public complementoMensaje?: string;
    public complementoEsFecha?: boolean;
    public fecha: string;
    public visto: boolean;
    public idChat?:number; 
}

File: Backend/TrackrAPI/TrackrAPI/Dtos/Notificaciones/NotificacionPacientePopOverDto.cs
namespace TrackrAPI.Dtos.Notificaciones;

public class NotificacionPacientePopOverDto{
    public int IdNotificacion {get; set;}
    public int IdTipoNotificacion { get; set; }
    public string Titulo {get; set; }
    public string Mensaje {get; set; }
    public string? ComplementoMensaje { get; set; }
    public DateTime FechaAlta {get; set; }
    public bool Visto{get; set; }
}